config.rb

```diff
@@ -1,12 +1,26 @@
 require 'breakpoint'
-require 'builder'
 require 'middleman-syntax'
 require 'susy'
-require 'typogruby'
 
+activate :automatic_image_sizes
+activate :autoprefixer do |config|
+  config.browsers = ['last 2 versions', 'Explorer >= 9']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.